### DISCUSSION TITLE
可以嘗試用IA Presenter來協作簡報

### DISCUSSION THREAD

**Discussion by bestian** [3dw/auto20-next#231] [Ideas]
@leechiuhui 

如果接下來我們要在自主學習促進會或是g0v 做出簡報的話，
協作一份好的簡報檔以便開會時呈現可能是一個好主意

這是使用MarkDown 為基礎的自動排版簡報編輯軟體IA Presenter簡介
想討論一下要不要共同使用這個東西來協作簡報，以替代與原本的Google簡報呢？

它有14天試用期可以玩玩看~

https://ia.net/presenter?ref=larrynote.com

**Comment by leechiuhui** [comment]
<img width="1512" alt="截圖 2024-10-12 上午11 01 52" src="https://github.com/user-attachments/assets/3117c7f6-d740-4ed1-a7cc-babd859096a9">
Teacher, I am trying  it now. Let's observe the results and see how it goes.Thank you~:)

**Reply by leechiuhui** [reply]
我在想應該是不需要去g0v提案，因為後來想想：要去找人來幫忙，我應該要先去幫忙人家才對。😃


![image](https://github.com/user-attachments/assets/b990b3ca-4e65-46d7-8ba1-f39ae79e130f)
